app/views/distributions/index.html.erb

```diff
@@ -24,18 +24,18 @@
             <% if @items.present? %>
               <div class="form-group col-lg-3 col-md-4 col-sm-6 col-xs-12">
                 <%= label_tag "by Item" %>
-                <%= collection_select(:filters, :by_item_id, @items || {}, :id, :name, class: "form-control", include_blank: true) %>
+                <%= collection_select(:filters, :by_item_id, @items || {}, :id, :name, {}, {class: "form-control"}) %>
               </div>
             <% end %>
             <% if @partners.present? %>
               <div class="form-group col-lg-3 col-md-4 col-sm-6 col-xs-12">
                 <%= label_tag "by Partner" %>
-                <%= collection_select(:filters, :by_partner, @partners || {}, :id, :name, class: "form-control", include_blank: true) %>
+                <%= collection_select(:filters, :by_partner, @partners || {}, :id, :name, {}, {class: "form-control"}) %>
               </div>
             <% end %>
             <div class="form-group col-lg-3 col-md-4 col-sm-6 col-xs-12">
             <%= label_tag "by Date" %>
-            <%= select(:filters, :interval, filter_intervals || [], class: "form-control", include_blank: true) %>
+            <%= select(:filters, :interval, filter_intervals || [], {}, {class: "form-control"}) %>
```

Comment:
Could you re-add `include_blank: true` to each of these? 

The filters are optional, so they need to have blank options in case you don't want to filter on that field.

Reply:
Sure! I will do that

Reply:
Man, it's done!